app/scripts/controllers/newOverview.js

```diff
@@ -835,7 +836,7 @@ function OverviewController($scope,
     var toUpdate = [
       overview.deploymentConfigs,
       overview.vanillaReplicationControllers,
-      overview.deploymentsByUID,
+      overview.deployments,
```

Comment:
Yes, and I had fixed it... I must not have added the file to the commit. Will fix.